components/camel-arangodb/src/main/java/org/apache/camel/component/arangodb/ArangoDbConfiguration.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.arangodb;
+
+import org.apache.camel.RuntimeCamelException;
+import org.apache.camel.spi.UriParam;
+import org.apache.camel.spi.UriParams;
+
+@UriParams
+public class ArangoDbConfiguration implements Cloneable {
+    private String database;
+    @UriParam
+    private String host;
+    @UriParam
+    private int port;
+    @UriParam(label = "security", secret = true)
+    private String user;
+    @UriParam(label = "security", secret = true)
+    private String password;
+    @UriParam
```

Comment:
Also label here are missing for the configurations, e.g: `producer`, `consumer` or `common` .. etc

Reply:
@zbendhiba can you please add labels here?

Reply:
yes , done !!